systems/identity/identity-bootstrap/src/main/kotlin/hs/kr/entrydsm/identity/config/security/JwtFilter.kt

```
@@ -0,0 +1,142 @@
+            val principal = verify(token)
+            SecurityContextHolder.getContext().authentication =
+                UsernamePasswordAuthenticationToken(principal, null, emptyList())
+            filterChain.doFilter(request, response)
+        } catch (exception: JwtValidationException) {
+            SecurityContextHolder.clearContext()
+            authenticationEntryPoint.commence(
+                request,
+                response,
+                BadCredentialsException("Invalid JWT", exception),
+            )
+        } catch (exception: RefreshTokenStoreUnavailableException) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)